assets/css/okta-plugin-a11y.css

```diff
@@ -22,7 +28,7 @@ button {
   color: #222!important;
 }
 #okta-sign-in.auth-container.main-container {
-  color: #222!important;
+  color: #767676!important;
```

Comment:
`#222` was too dark. The contrast between normal text and "subtle" text was not perceivable for sighted users